### PR TITLE
Nick Edwards JS Basics Challenge Submission

### PR DESCRIPTION
**Pull request by nickedwards109**

### CODE REVIEW

File: lib/triangle.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    return false;
+  }
+}
+
+function verifyPythagoreanTriple(side1, side2, side3) {
+  if ( (Math.pow(side1, 2) + Math.pow(side2, 2) ) == Math.pow(side3, 2) ) {
+    return true;
+  } else {
+    return false;
+  }
```
Comment (laurenfazah) [inline lib/triangle.js]: Nice helper method!

File: lib/averages.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function total(data) {
+  var sum = 0;
+  for(i = 0; i < data.length; i++) {
+    sum += data[i];
+  }
+  return sum;
```
Comment (laurenfazah) [inline lib/averages.js]: Does this feel like any Ruby enumerable you know? Hint, it exists in JS too :)

**Comment by laurenfazah** [comment]
Great git workflow, Nick. Made it really easy for me to follow your process throughout the challenge.

Your solutions were strong and easily readable. I liked seeing some helper methods here and there. As we develop stronger JS habits, we'll be pushing you to do more of this. 

I love that I saw you're using some ES6 syntax. I'd suggest that you keep consistency with that, though. For example, if using ES6, you can pretty much always replace `var` with `const` (or `let` if that variable's value will be changing).

**Comment by nickedwards109** [comment]
Thanks, @laurenfazah!

File: lib/averages.js (hunk 2 above)
Comment (nickedwards109) [inline lib/averages.js]: aha, it's reduce!

**Pull request closed without merging**